### ISSUE TITLE
Woocommerce multilingual error on activate plugin

### ISSUE THREAD

**Issue by dudinio** [timber/timber#2054]
Base WPML (v4.2.7.1) with WPML String Translation (v2.10.5.1), WPML Translation Management(2.8.6.2), WPML Media(v2.5.2), Advanced Custom Fields Multilingual (v1.4.0) and Yoast SEO Multilingual(v1.0.1) - works fine without bugs.

Timber version 1.8.4 

```
Fatal error: Uncaught TypeError: Argument 1 passed to Twig_Node_Expression_Filter::__construct() must be an instance of Twig_NodeInterface, instance of Twig_Node_Expression_Name given, called in /Users/lukasz/Desktop/git/off-district/web/app/plugins/woocommerce-multilingual/vendor/twig/twig/lib/Twig/ExpressionParser.php on line 483 and defined in /Users/lukasz/Desktop/git/off-district/web/app/plugins/woocommerce-multilingual/vendor/twig/twig/lib/Twig/Node/Expression/Filter.php:14 Stack trace: #0 /Users/lukasz/Desktop/git/off-district/web/app/plugins/woocommerce-multilingual/vendor/twig/twig/lib/Twig/ExpressionParser.php(483): Twig_Node_Expression_Filter->__construct(Object(Twig_Node_Expression_Name), Object(Twig_Node_Expression_Constant), Object(Twig\Node\Node), 14, NULL) #1 /Users/lukasz/Desktop/git/off-district/web/app/plugins/woocommerce-multilingual/vendor/twig/twig/lib/Twig/ExpressionParser.php(466): Twig_ExpressionParser->parseFilterExpressionRaw(Object(Twig_Node_Expression_Name)) #2 /Users/lukasz/Desktop/git/off-di in /Users/lukasz/Desktop/git/off-district/web/app/plugins/woocommerce-multilingual/vendor/twig/twig/lib/Twig/Node/Expression/Filter.php on line 14
The site is experiencing technical difficulties. Please check your site admin email inbox for instructions.
```

**Comment by palmiak** [comment]
What I discovered is the fact I can't replicate it on php 7.3

**Comment by jarednova** [comment]
@dudinio this is an issue based on a conflict with WPML's Twig version and our own. I've spoken with the developer and it will be resolved in 4.2.8 — expected very soon
